### ISSUE TITLE
Microphone/Speaker Problem

### ISSUE THREAD

**Issue by xTheCodingChicken** [alexa/alexa-avs-sample-app#899]
Hey,
I have a problem with my microphone/speaker. When I'm trying to activate Alexa by pressing the button in the java program, I'm getting this error: https://pastebin.com/gtM7KpC1 and nothing happens when I say Alexa. I'm using this microphone: https://www.amazon.de/gp/product/B00XA01IQC/ref=oh_aui_detailpage_o03_s01?ie=UTF8&psc=1 !
(sorry for the German link, this product doesn't seem to be available in the US version of Amazon...)

**Comment by dhpp** [comment]
Hi there,

We have seen this occur in the past if you are using the wake word agent, which shares the microphone with the Java sample app.  In that case you need to make sure your setup is correct, per our instructions.  The wake word agent exists as a different process if you are using it.  Can you advise if this is what you're trying to do?  Thanks.

**Comment by xTheCodingChicken** [comment]
Hi,
so I first started this project with the tutorial of a German YouTuber. Today I completely reinstalled Raspbian and followed the instructions in the wiki and I got it to play the Bling sound when you say "Alexa" but just one time and now she isn't even reacting to the button in the program anymore. I just get this error in the 3. Terminal ( wake word engine: https://pastebin.com/EFegnRwP ) and this error in the 2. terminal ( the exec:exec thingy: https://pastebin.com/r97EJtHb ) @dhpp Yes I'm trying to use the wake word engine.
